tests/import_library.def

```diff
@@ -11,6 +11,53 @@ EXPORTS
    ReexportedFunc=OtherModule.OtherName
    ReexportedViaOrd=OtherModule.#42
    FuncWithImportName==ImportName
-   ?CppFunc@SingleAt
-   ?CppFunc@@DoubleAt
-   ?CppFunc@@@TripleAt
+
+
+   ; These are Arm64EC mangled names, since we currently don't support converting
+   ; the non-Arm64EC format to Arm64EC. They are taken from ManglerTest.cpp
```

Comment:
The .def file could contain the non-arm64ec mangled names, right?

Reply:
Correct: we won't be able to handle C++ mangled names that haven't also been Arm64EC mangled. Currently it's broken anyway (the previous heuristic was wrong).